ecma402_traits/src/relativetimeformat.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt;
+
+/// Contains the API configuration as prescribed by [ECMA 402][ecma].
+///
+///    [ecma]: https://www.ecma-international.org/publications/standards/Ecma-402.htm
+///
+/// The meaning of the options is the same as in the similarly named
+/// options in the JS version.
+///
+/// See [Options] for the contents of the options.  See the [RelativeTimeFormat::try_new]
+/// for the use of the options.
+pub mod options {
+    /// Whether to use numeric formatting in the display, like "1 day ago".
+    #[derive(Debug, Clone, PartialEq)]
+    pub enum Numeric {
+        /// Always use numeric formatting.
+        ALWAYS,
```

Comment:
If I'm not mistakes cargo clippy recommends Always, not ALWAYS

Reply:
Done.